tune_sklearn/tune_basesearch.py

```diff
@@ -433,8 +433,17 @@ def _can_early_stop(self):
             bool: if the estimator can early stop
 
         """
-        return (hasattr(self.estimator, "partial_fit")
-                and callable(getattr(self.estimator, "partial_fit", None)))
+        sklearn_module = ".".join(
+            getattr(self.estimator, "__module__", None).split(".")[:-1])
```

Comment:
This doesn't seem like it'll work for all cases. If it returns "none", you'll call None.split, which will fail.